hazelcast/src/main/java/com/hazelcast/map/impl/mapstore/writebehind/WriteBehindStore.java

```diff
@@ -150,13 +163,29 @@ public Object add(Data key, Object value, long expirationTime, long now) {
         DelayedEntry<Data, Object> delayedEntry
                 = DelayedEntries.createDefault(key, value, expirationTime, now, partitionId);
 
-        add(delayedEntry);
+        add(delayedEntry, transactionId);
 
         return value;
     }
 
-    public void add(DelayedEntry<Data, Object> delayedEntry) {
-        writeBehindQueue.addLast(delayedEntry);
+    @Override
+    public void addForcibly(DelayedEntry<Data, Object> delayedEntry) {
```

Comment:
It would be useful to have Javadoc when the forced method is used?

Reply:
added javaDoc.